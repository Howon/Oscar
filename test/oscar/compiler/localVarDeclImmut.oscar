def main() => unit = {
	int i = 17;
	Println(i);

	char c = 'c';
	Println(c);

	double d = 42.0;
	Println(d);	

	bool b = true;
	Println(b);

	string s = "brug";
	Println(s);
}
