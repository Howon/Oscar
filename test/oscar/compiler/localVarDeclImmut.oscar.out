17
c
42.000000
true
brug
